frontend/src/components/DataCard/CardData.jsx

```diff
@@ -1,9 +1,17 @@
 import React from "react";
+import {
+    Button,
+    Tooltip,
+    OverlayTrigger
+} from "react-bootstrap";
 
-export default function CardData({ card }) {
+export default function CardData({ card, onSkip }) {
     return (
         <div className="cardface-info">
-            <h2>Card {card.id + 1}</h2>
+            <div className="card-title">
+                <h2>Card {card.id + 1}</h2>
```

Comment:
So I believe this may not have been discussed before this MR was made, but since we don't have a progress bar anymore, the card numbers are probably more confusing than anything else. So we probably want to drop those too.